openedx/core/djangoapps/catalog/tests/mixins.py

```diff
@@ -9,6 +9,7 @@ class CatalogIntegrationMixin(object):
         'enabled': True,
         'internal_api_url': 'https://catalog-internal.example.com/api/v1/',
         'cache_ttl': 0,
+        'service_username': 'lms_catalog_service_user'
```

Comment:
This username is used in like 20 places including existing migrations, I think we should be consistent at this point